From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@AzureDocsBot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jodoglevy commented:
Please ignore, testing for hackadoc

alexkarcher-msft committed:
Updating supported services

alexkarcher-msft committed:
Updated TOC to include external table

carolinacmoravia committed:
C52861: Broken bookmark: [Availabilities for HDInsight cluster] [Access blobs using Azure PowerShell]

@jgao, 
Could you review the reported bookmarks? They are not working.
Many thanks for your time in advance.

kavyako committed:
reverse proxy diagnostics

kavyako committed:
acrolix fixes

Juliako committed:
updated

carolinacmoravia committed:
M6734: Typo mistake: Use the following code to connect and read the data using a DELETE SQL statement. 

Correcting typo mistake. It should "read" instead of "delete."

Commit:
Create azure-operational-security-best-practices.md

Commit:
Create azure-service-fabric-security-overview.md

Commit:
Create azure-service-fabric-security-best-practices.md

Commit:
Create azure-service-fabric-security-checklist.md

Commit:
Update azure-service-fabric-security-checklist.md

Commit:
Update azure-service-fabric-security-checklist.md

Commit:
Update azure-service-fabric-security-checklist.md

Commit:
Update azure-service-fabric-security-checklist.md

Commit:
Update azure-service-fabric-security-best-practices.md

Commit:
Update azure-operational-security-best-practices.md

Commit:
Update azure-operational-security-best-practices.md

pchoudhari committed:
Add active learning

pchoudhari committed:
Add active learning image

pchoudhari committed:
Update toc.yml

pchoudhari committed:
Update active-learning-qnamaker.md

carolinacmoravia committed:
M6840: Missing preposition: "Site Recovery automatically discovers VMs located and vSphere ESXi hosts, and/or managed by vCenter servers.

@rayne-wiselman, 
It seems that the setence requeires a preposition. Could you check if the proposed file change is correct?
Many thanks in advance.

jawache committed:
Added note that the sample is in Python

This doc is going to be linked into any of the language specific dev centres, so we are adding in a note to not confuse the readers and let them know that it works regardless of language.

jawache committed:
Update container-service-kubernetes-walkthrough.md

jawache committed:
Moved key info up the page to clarify

Explain that the following kubernetes config uses "pre-created" container images. Those container images are what contain the sample application which is what we are deploying. Then follow on to explain to the user how they can create their own container images.

This info was at the bottom,

jawache committed:
Fix for acrolynx

pchoudhari committed:
Update active-learning-qnamaker.md

pchoudhari committed:
Update active-learning-qnamaker.md

rnagpal committed:
Release Notes for .NET SDK 1.16.1

@mimig1 @arramac : Please review and I'll sign off.

rnagpal committed:
Release Notes for .NET Core SDK 1.4.1

@mimig1 @arramac : Pease review and I'll sign off.

pchoudhari committed:
Update active-learning-qnamaker.md

pchoudhari committed:
Update toc.yml

rothja committed:
Update of provisioning topic for freshness and clarity

jawache committed:
Changed wording as per review feedback

rothja committed:
Fixing validation error

jawache committed:
Removed trailing whitespace

barclayn committed:
adding Sunil's changes

barclayn committed:
modifying to comply with acrolynx

rothja committed:
Removing old experiment article that was never used

barclayn committed:
fixes for Acrolynx continued

barclayn committed:
removing the space to fix the metadata problem

barclayn committed:
more formatting fun

rnagpal committed:
Update documentdb-sdk-dotnet.md

rnagpal committed:
Update documentdb-sdk-dotnet.md

rnagpal committed:
Update documentdb-sdk-dotnet-core.md

parakhj committed:
Minor syntax updates

botatoes committed:
Add related links

botatoes committed:
Add related links

parakhj committed:
Merge branch 'master' into master

botatoes committed:
Add related links

botatoes committed:
Update related links

botatoes committed:
Update app-insights-usage-send-user-context.md

botatoes committed:
Update app-insights-usage-retention.md

botatoes committed:
Update app-insights-usage-segmentation.md

botatoes committed:
Update app-insights-usage-workbooks.md

botatoes committed:
Update usage-funnels.md

botatoes committed:
Update usage-funnels.md

kavyako committed:
Incorporate review feedback

kavyako committed:
update reverse proxy default settings

kavyako committed:
doc review feedback

kavyako committed:
Update link

venkatgct committed:
Make tenant name mandatory

Commit:
Merge pull request #18920 from venkatgct/master

Make tenant name mandatory

botatoes committed:
Update app-insights-usage-workbooks.md

mmacy committed:
[acs] squash redirects files w/Gauntlet

botatoes committed:
Update app-insights-usage-workbooks.md

MGoedtel committed:
added specific steps for obtianing OMS ID and Key

MGoedtel committed:
revised steps for getting key and id

Commit:
Merge pull request #18907 from botatoes/bofu-patch1

Add related link to usage docs

Commit:
Merge pull request #18923 from MGoedtel/UpdateContainers882017

added specific steps for obtianing OMS ID and Key

JasonWHowell committed:
Usability study stuff

v-thepet committed:
it never ends

kavyako committed:
doc review feedback

curtand committed:
add link to Azure DNS

botatoes committed:
Update app-insights-usage-overview.md

botatoes committed:
Update app-insights-usage-retention.md

GitHubber17 committed:
Merge pull request #18924 from mmacy/acs-redirect-squash-03

[acs] squash redirect files w/Gauntlet

botatoes committed:
Update app-insights-usage-segmentation.md

botatoes committed:
Update app-insights-usage-workbooks.md

ktoliver committed:
Merge pull request #17903 from kavyako/rp_update

reverse proxy diagnostics

Commit:
Merge pull request #18856 from rnagpal/patch-9

Release Notes for .NET Core SDK 1.4.1

Juliako committed:
Update video-indexer-use-apis.md

Commit:
Merge pull request #18855 from rnagpal/patch-8

Release Notes for .NET SDK 1.16.1

Juliako committed:
Update video-indexer-output-json.md

Commit:
Merge pull request #18905 from parakhj/master

Minor syntax updates

Commit:
vm faq fix

Commit:
Merge pull request #18932 from Juliako/patch-8

Update video-indexer-use-apis.md

Juliako committed:
Update video-indexer-output-json.md

Commit:
Merge pull request #18931 from botatoes/bofu-patch1

Update related links to be consistent

Commit:
fixing links

Commit:
Merge pull request #18933 from Juliako/patch-10

Update video-indexer-output-json.md

Commit:
Merge pull request #18927 from JasonWHowell/PostgresUsability

Postgresql: Usability study stuff

ktoliver committed:
Merge pull request #18525 from Juliako/Redactor

updated

curtand committed:
rewrite procedures for accuracy

Commit:
Merge pull request #18631 from carolinacmoravia/patch-303

M6734: Typo mistake: Use the following code to connect and read the d…

Commit:
Merge pull request #18934 from v-thepet/fixlinks

production bug: it never ends

curtand committed:
acrolinx 01

GitHubber17 committed:
Merge pull request #18794 from pchoudhari/master

Add active learning docs

barbkess committed:
change images from gif to png

GitHubber17 committed:
Merge pull request #18929 from curtand/class0808

add link to Azure DNS

GitHubber17 committed:
Merge pull request #18943 from barbkess/ron

change images from gif to png

ecfan committed:
Upload reviewed OMS topic and art

ecfan committed:
Update ToC

ecfan committed:
Fix Acrolinx issues

ecfan committed:
Fix broken link

MightyPen committed:
Update sql-database-connect-query-php.md

Installation links, into http://aka.ms/sqldev, wrongly point to 'node/' here in this 'php/' .md file.  Fixing.

Commit:
Merge pull request #18949 from MightyPen/patch-2

Update sql-database-connect-query-php.md

ecfan committed:
Fix broken link

MightyPen committed:
Update sql-database-connect-query-python.md

Ubuntu and Windows links, into http://aka.ms/sqldev, wrongly pointed to 'node/' instead of to 'python/'.  Now fixing.
MightyPen = genemi.

Commit:
Merge pull request #18950 from MightyPen/patch-3

Update sql-database-connect-query-python.md

Commit:
Update data-lake-analytics-u-sql-get-started.md

ecfan committed:
Removed topic link

genlin committed:
add text about signin issue

genlin committed:
correct typo

Commit:
Merge pull request #18953 from genlin/65107

storage-explorer-troubleshooting.md

genlin committed:
add boot errors

genlin committed:
change the location of the boot error

Commit:
Merge pull request #18952 from saveenr/patch-6

Update data-lake-analytics-u-sql-get-started.md

rothja committed:
Simplified connect and made other fixes

mumian committed:
refresh the Avro article

mumian committed:
acrolinx

rothja committed:
Added missing border on image

Commit:
Merge pull request #18959 from mumian/avro0809

Avro0809

mumian committed:
Merge branch 'patch-270' of https://github.com/carolinacmoravia/azure-docs-pr into blobstorage0809

mumian committed:
fix the two broken links

mumian committed:
update

itechedit committed:
Merge pull request #18695 from unifycloud/patch-1

Create azure-operational-security-best-practices.md

itechedit committed:
Merge pull request #18697 from unifycloud/patch-2

Create azure-service-fabric-security-overview.md

v-shils committed:
Merge pull request #18954 from genlin/65107

Virtual-machines-common-boot-diagnostics.md

Commit:
Merge pull request #18798 from carolinacmoravia/patch-304

M6840: Missing preposition: "Site Recovery automatically discovers VM…

itechedit committed:
Merge pull request #18698 from unifycloud/patch-3

Create azure-service-fabric-security-best-practices.md

itechedit committed:
Merge pull request #18699 from unifycloud/patch-4

Create azure-service-fabric-security-checklist.md

carolinacmoravia committed:
M6840: Missing preposition: "Site Recovery automatically discovers VM… #18798

@rayne-wiselman,
File modified as indicated. 
Thank you.

TomShinder committed:
Update azure-database-security-overview.md

fixed small typo

v-shils committed:
Merge pull request #18946 from ecfan/oms-updates

New topic and art for logic app monitoring in Operations Management Suite

Commit:
Merge pull request #18963 from carolinacmoravia/patch-309

M6840: Missing preposition: "Site Recovery automatically discovers VM…

mumian committed:
update a link

Commit:
Merge pull request #18964 from TomShinder/patch-23

Update azure-database-security-overview.md

Commit:
Merge pull request #18961 from mumian/blobstorage0809

Blobstorage0809

itechedit committed:
Merge pull request #18870 from rothja/provisionupdates

Update of provisioning topic for freshness and clarity

itechedit committed:
Merge pull request #18882 from barclayn/sunil-updates

adding Sunil's changes

alexkarcher-msft committed:
Brief formatting updates

itechedit committed:
Merge pull request #18935 from singhkays/master

Migrate vm faq fix

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #18967 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #14422 from alexkarcher-msft/master

Updated experimental Functions binding docs

Commit:
Merge pull request #18809 from jawache/patch-1

Kubernetes for Linux Quickstart
